notebooks/examples/create_a_custom_suite.ipynb

```diff
@@ -0,0 +1,759 @@
+{
```

Comment:
<p>Is that the place to mention the option of "start with predefined suite and slightly modify it to your needs"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/237/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Can we remove &amp; add checks to built-in suites?</p><p>If so I think we should add that, but also a part that describes how to do that.</p>

Reply:
<p>yes it's possible</p>

Reply:
<p>Ok so I'll add a short description, with a todo for the example</p><p> </p>